### PR TITLE
dagger 0.2.7

### PR DESCRIPTION
**Pull request by solidnerd**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by solidnerd** [comment]
They had a behaviour change in the `dagger do` command between https://github.com/dagger/dagger/compare/v0.2.6...v0.2.7 Check for `cmd/dagger/cmd/do.go`. I think we need to change the assertion here a little bit.

File: Formula/dagger.rb [hunk 1]
```diff
@@ -45,6 +45,6 @@ def install
     assert_predicate testpath/"cue.mod/module.cue", :exist?
 
     output = shell_output("#{bin}/dagger do test 2>&1", 1)
-    assert_match(/(denied while trying to|Cannot) connect to the Docker daemon/, output)
+    assert_match(/failed to load plan: no CUE files in /, output)
```
Comment (SMillerDev) [inline Formula/dagger.rb:48]: But the command above actually checks if they exist?

Comment (solidnerd) [reply Formula/dagger.rb]: @SMillerDev In General the command above checks only a if the dependency file was throug `dagger project update` created. With the `dagger do` command you need to have a separated file like `hello.cue` with action called test.


The behaviour of the command has changed between the patch version. I did a test with a direct repo install locally on my machine

as pre requisiuite I checked out the repository

```console
git clone  https://github.com/dagger/dagger
```

## 0.2.5

```console
git checkout v0.2.5
make install 
```

```
mkdir -p /tmp/dagger-0.2.5
cd /tmp/dagger-0.2.5
```

```console
dagger version
dagger devel (7d2f279c) darwin/amd64
```

```console
dagger project init
dagger project update

```

```console
dagger do test 2>&1
```

```console
1:54PM ERR failed to run docker: exit status 1    output=Client:
 Context:    default
1:54PM ERR failed to run docker: exit status 1    output=Client:
 Context:    default
 Debug Mode: falseuildx (Docker Inc., v0.8.1)
 Plugins:: Docker Compose (Docker Inc., v2.3.3)
  buildx: Docker Buildx (Docker Inc., v0.8.1)
  compose: Docker Compose (Docker Inc., v2.3.3)
  scan: Docker Scan (Docker Inc., v0.17.0)
ERROR: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
Server:pretty printing info
ERROR: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
errors pretty printing info
1:54PM FTL unable to create client: exit status 1
```

## 0.2.6

```console
git checkout v0.2.6
make install 
```

```
mkdir -p /tmp/dagger-0.2.6
cd /tmp/dagger-0.2.6
```

```console
dagger version
dagger devel (e2c2213a) darwin/amd64
```

```console
dagger project init
dagger project update

```

```console
dagger do test 2>&1
```

```console
1:59PM ERR failed to run docker: exit status 1    output=Client:
 Context:    default
1:59PM ERR failed to run docker: exit status 1    output=Client:
 Context:    default
 Debug Mode: falseuildx (Docker Inc., v0.8.1)
 Plugins:: Docker Compose (Docker Inc., v2.3.3)
  buildx: Docker Buildx (Docker Inc., v0.8.1)
  compose: Docker Compose (Docker Inc., v2.3.3)
  scan: Docker Scan (Docker Inc., v0.17.0)
ERROR: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
Server:pretty printing info
ERROR: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
errors pretty printing info

1:59PM FTL unable to create client: exit status 1
```

## 0.2.7

```console
git checkout v0.2.7
make install 
```

```
mkdir -p /tmp/dagger-0.2.7
cd /tmp/dagger-0.2.7
```

```console
dagger version
dagger devel (18c19174) darwin/amd64
```

```console
dagger project init
dagger project update

```

```console
dagger do test 2>&1
```

```console
failed to load plan: no CUE files in .


Usage:
  dagger do [flags]


Flags:
      --cache-from stringArray   External cache sources (eg. user/app:cache, type=local,src=path/to/dir)
      --cache-to stringArray     Cache export destinations (eg. user/app:cache, type=local,dest=path/to/dir)
  -h, --help                     help for do
      --no-cache                 Disable caching
  -p, --plan string              Path to plan (defaults to current directory) (default ".")
      --platform string          Set target build platform (requires experimental)
  -w, --with stringArray
Global Flags:
      --experimental        Enable experimental features
      --log-format string   Log format (auto, plain, tty, json) (default "auto")
  -l, --log-level string    Log level (default "info")
```

**Comment by solidnerd** [comment]
I tried now to use the new template function to get the old behaviour back without changing the test.

**Comment by solidnerd** [comment]
@SMillerDev  Tests are finally passing now without a test change 🎉

**Review by dawidd6 — APPROVED** [review]

**Pull request closed without merging**